### DISCUSSION TITLE
Hello there, someone kindly help . i keep on getting errors after uploading my school documents so that i get student developer pack . the error says am not near the location of our school, and the errors keeps on asking me to upload more documents to show am a student . i have uploaded the documents  but still  the errors keeps on passisting , another point to note in my school we are on holiday but am not that far vrom our school its just few kilometers from my home .

### DISCUSSION THREAD

**Discussion by Muiruri590** [community/community#123660] [GitHub Education]
### Hi everyone,

student developer pack fails approval.
i have applied for student developer pack but i keep on getting errors, saying that am not near my school , and the other says i should upload more documents to approve am student . point to note we are currently on holiday but am not that far from our school its just few kilometers from where i am . 
what should i do , kindly help

**Comment by InWamos** [comment]
> and the errors keeps on asking me to upload more documents to show am a student 

I faced the same issue. You have to upload both your study contract (where the timespan is stated) and your student ID
